test/integration/basic_test.go

```diff
@@ -177,15 +260,22 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`Minishift should have a valid IP address`, m.shouldHaveAValidIPAddress)
 	s.Step(`executing "minishift ([^"]*)"`, m.executingCommand)
 	s.Step(`executing "oc ([^"]*)" retrying (\d+) times with wait period of (\d+) seconds$`, m.executingRetryingTimesWithWaitPeriodOfSeconds)
+	s.Step(`([^"]*) of command "minishift ([^"]*)" is equal to "([^"]*)"`, m.commandReturnEquals)
+	s.Step(`([^"]*) of command "minishift ([^"]*)" contains "([^"]*)"`, m.commandReturnContains)
 	s.Step(`executing "oc ([^"]*)`, m.executingOcCommand)
 	s.Step(`([^"]*) should contain ([^"]*)`, m.commandReturnShouldContain)
 	s.Step(`([^"]*) should contain`, m.commandReturnShouldContainContent)
 	s.Step(`([^"]*) should equal ([^"]*)`, m.commandReturnShouldEqual)
 	s.Step(`([^"]*) should equal`, m.commandReturnShouldEqualContent)
 	s.Step(`([^"]*) should be empty`, m.commandReturnShouldBeEmpty)
+	s.Step(`response.?([^"]*)? at "([^"]*)" contains "([^"]*)"`, m.getUrlResponse)
```

Comment:
"response" is a bit vague.  I also find the second capture group a bit odd. What else is there expect "OpenShift". How is this going to be used moving forward for non OpenShift URLs?